### PR TITLE
Revisions to glyma.Chen_Fang_2019.yml on 26 April 2024

### PR DESCRIPTION
**Pull request by ScottKalberer**
I've updated the gene_function_registry file glyma.Chen_Fang_2019.yml, created the new branch glyma.Chen_Fang_2019, and pushed everything to the GitHub origin. This entity was originally created by Greg Murrell.  I'm requesting review of this YML gene function file, merge of the changes into main branch, and subsequent deletion of this temporary branch on GitHub.

### CODE REVIEW

**Review by StevenCannon-USDA — DISMISSED** [review]

File: Glycine/max/studies/glyma.Chen_Fang_2019.yml
Comment (StevenCannon-USDA) [inline Glycine/max/studies/glyma.Chen_Fang_2019.yml:1]: Generally looks good, but line 20 generates a yamllint error. You can fix it by removing one of the spaces after `phenotype_synopsis:`

**Review by StevenCannon-USDA — CHANGES_REQUESTED** [review]
Please remove the extraneous space after `phenotype_synopsis:`
(picky, but triggers an error in yamllint)

File: Glycine/max/studies/glyma.Chen_Fang_2019.yml
Comment (ScottKalberer) [reply Glycine/max/studies/glyma.Chen_Fang_2019.yml:1]: I have removed the extraneous space following the phenotype_synopsis:.  However, I could not get yamllint to produce an error message using this file, regardless of how many extra spaces I added.  Perhaps there's a setting on their website that I need to adjust to see this problem?

**Comment by ScottKalberer** [comment]
Hi Steven, I removed the unwanted space from the gene function file Chen_Fang_2019.yml.  Please approve this pull request and merge if you are satisfied.

**Review by StevenCannon-USDA — APPROVED** [review]

**Comment by StevenCannon-USDA** [comment]
The latest changes seem to have taken care of the yamllint complaints.

**Pull request merged**